GitOut/Features/Git/Stage/StatusChangeViewModel.cs

```diff
@@ -30,7 +30,7 @@ private StatusChangeViewModel(GitStatusChange model, StatusChangeLocation locati
                 Status = status.Value;
                 IconResourceKey = status switch
                 {
-                    GitModifiedStatusType.Added => "FilePlus",
+                    GitModifiedStatusType.Added => "FileEdit",
```

Comment:
With this change both modified and added files will have the same icon, is that intentional? 🤔